android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -497,12 +497,8 @@ public void focus()
 	{
 		super.focus();
 		if (tv != null) {
-			if (proxy.hasProperty(TiC.PROPERTY_EDITABLE)
-				&& !(TiConvert.toBoolean(proxy.getProperty(TiC.PROPERTY_EDITABLE)))) {
-				TiUIHelper.showSoftKeyboard(tv, false);
-			} else {
-				TiUIHelper.showSoftKeyboard(tv, true);
-			}
+			final boolean editable = proxy.getProperties().optBoolean(TiC.PROPERTY_EDITABLE, false);
```

Comment:
The `getProperties()` method can return `null` when an instance of this class is used by `Ti.UI.SearchBar`. In this case, an instance of this class does not have a proxy. See here...
https://github.com/appcelerator/titanium_mobile/pull/10848